### PR TITLE
Total messages and users

### PR DESCRIPTION
**Pull request by amruta-jv**

### CODE REVIEW

File: controller/sandbox.js [hunk 1]
```diff
@@ -2,105 +2,227 @@ const Sandbox = require('../models/sandbox');
...
-    });
-  
-    sandbox.save().then((doc) => {
-      res.send(doc);
+  var payload = req.body;
+  var sandbox = new Sandbox({
+    sandbox_id: req.body.sandbox_id,
+    bot_details: {
+      state: req.body.state,
+      bot_id: req.body.bot_id
```
Comment (amruta-jv) [inline controller/sandbox.js]: Can we read above fields from `payload` instead `req`.

File: index(copy).js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        cursor.forEach((doc,err)=> {
+            data.push(doc);
+        }, () =>{
+            if(!err){
+                res.send({totalRecords: data.length, data: data});
+            }
+        });
+        client.close();
+    });
+});
```
Comment (amruta-jv) [inline index(copy).js]: Why we added this file again?

File: models/sandbox-local.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    },    
+    updated_on: { type: Date,
+        default: Date.now
+    }
+},
+{
+    collection:"sandbox_interactions"
+});
+
+module.exports = mongoose.model('SandboxInteractions', inter_schema);
```
Comment (amruta-jv) [inline models/sandbox-local.js]: Why this file?

File: routes/index.js [hunk 4]
```diff
@@ -9,9 +9,11 @@ router.get('/', function (req, res) {
...
 });
 
-//router.route('/search').post(controller.search);
-router.route('/sandboxinteractions').get(sandboxController.getAll);
-router.route('/sandboxinteractions').post(sandboxController.insert);
-router.route('/totalconversations').get(sandboxController.getConversationCount);
-    
+router.route('/insert').post(sandboxController.insert);
+router.route('/getAll').get(sandboxController.getAll);
+router.route('/getConversationCount').get(sandboxController.getConversationCount);
```
Comment (amruta-jv) [inline routes/index.js]: Why above three urls are changed?

File: index(copy).js (hunk 2 above)
Comment (abhay-mandal) [reply index(copy).js]: will remove that.

File: models/sandbox-local.js (hunk 3 above)
Comment (abhay-mandal) [reply models/sandbox-local.js]: removed thanks

File: controller/sandbox.js (hunk 1 above)
Comment (abhay-mandal) [reply controller/sandbox.js]: ok will do it

File: .gitignore [hunk 5]
```diff
@@ -1,2 +1,3 @@
 /node_modules
+index(copy).js
```
Comment (amruta-jv) [inline .gitignore]: Let's remove this as it's not present

**Pull request merged**